test/e2e/memcached_test.go

```diff
@@ -113,6 +117,12 @@ func TestMemcached(t *testing.T) {
 		t.Fatalf("error: %v\nCommand Output: %s\n", err, string(cmdOut))
 	}
 
+	// Set replicas to 2 to test leader election. In production, this should almost always be set to 1.
+	cmdOut, err = exec.Command("sed", "-i", "-e", "s/replicas: 1/replicas: 2/", "deploy/operator.yaml").CombinedOutput()
```

Comment:
We should avoid using `sed` in code for portability reasons (see the note in this [user guide section](https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md#1-run-as-a-deployment-inside-the-cluster)). This could be `ioutil.ReadFile() -> bytes.Replace(...) -> ioutil.WriteFile()`.